### ISSUE TITLE
Thief can Pickpocket Bandit multiple times, using Bandit's own move

### ISSUE THREAD

**Issue by OtherworldBob** [TerryCavanagh/diceydungeons.com#218]
When fighting the Bandit, the Thief can borrow and use the Bandit's Pickpocket move multiple times. This allows you to accumulate a reasonable amount of gold if the Bandit doesn't manage to RunAway very quickly, and you get lucky with the Pickpocket move being chosen to be borrowed frequently.

Not sure if this affects all single-use-per-battle items or not. Apologies if this has been reported before.

**Comment by joeld42** [comment]
Yeah I ran into this too. With the old map it was OK because after he fled he was gone so you might end up with some extra gold and just feel lucky, but now that he stays on the map you can just keep visiting him and grind infinite monies (I was getting about +2 per battle).

**Comment by TerryCavanagh** [comment]
This is almost fixed, I think, but need to check this case:

>regular pickpocket works fine, pickpocket- is the bugged one

**Comment by ncrecc** [comment]
> This is almost fixed, I think, but need to check this case:
> 
> > regular pickpocket works fine, pickpocket- is the bugged one

If that's the case, this is a dupe of #424
